.eslintrc.js

```diff
@@ -1,6 +1,25 @@
-const eslintConfig = require('@hashicorp/platform-cli/config/.eslintrc.js')
 module.exports = {
```

Comment:
Copied our current settings from dev portal to align with our standard lint rules.